test/functional/product/create_test.rb

```diff
@@ -0,0 +1,38 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+require File.join(File.dirname(__FILE__), '../organization/organization_helpers')
+require File.join(File.dirname(__FILE__), '../sync_plan/sync_plan_helpers')
+require File.join(File.dirname(__FILE__), '../product/product_helpers')
+
+describe "create a product" do
```

Comment:
Do products need to be associated with an organization? It's not clear from these tests, but it wouldn't hurt to test the failing case (i.e. `hammer product create --name product1` should fail if org is required).